data/userRepository.js

```diff
@@ -0,0 +1,20 @@
+class UserRepository {
+  constructor(data){
+    this.usersData = data
+  }
+  retrieveUserInfo(id){
+    let userInfo = this.usersData.filter(user =>
+    user.id == id)
```

Comment:
I think we can revisit how to approach the loosely equal to be a strict equal by turning the `user.id` into a number and use strictly equal. It might be a number??? let's talk about it